cybergis_compute_client/CyberGISCompute.py

```diff
@@ -380,6 +381,18 @@ def list_info(self, list_maintainer=False, list_container=False):
             self.list_maintainer()
 
     def create_job_by_ui(self, defaultJob="hello_world", defaultDataFolder="./", defaultRemoteResultFolder=None):
+        """
+        Displays the job submission UI
```

Comment:
I think you need a blank line before "Args:" and "Returns:" for this to render correctly with the autodocs.

Reply:
Like this? If so, do you want me to change the rest of the file too? I also noticed that there are some formatting inconsistencies (missing colons, capitalization, etc.) in this file. I can clean those up as well.

![image](https://user-images.githubusercontent.com/33166821/171509278-eada2e61-f50a-4ab2-a9ed-3fd9a7b69c3a.png)